packages/framework/aqueduct/src/container-runtime-factories/containerRuntimeFactoryWithDefaultDataStore.ts

```diff
@@ -51,18 +50,26 @@ export class ContainerRuntimeFactoryWithDefaultDataStore extends BaseContainerRu
 		defaultFactory: IFluidDataStoreFactory;
 		registryEntries: NamedFluidDataStoreRegistryEntries;
 		dependencyContainer?: IFluidDependencySynthesizer;
-		/** @deprecated Will be removed in future major release. Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md */
+		/** @deprecated Will be removed once Loader LTS version is "2.0.0-internal.7.0.0". Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md */
 		requestHandlers?: RuntimeRequestHandler[];
 		runtimeOptions?: IContainerRuntimeOptions;
 		provideEntryPoint?: (runtime: IContainerRuntime) => Promise<FluidObject>;
 	}) {
 		const requestHandlers = props.requestHandlers ?? [];
 		const provideEntryPoint = props.provideEntryPoint ?? getDefaultFluidObject;
 
+		const getDefaultObject = async (
+			req: IRequest,
+			runtime: IContainerRuntime,
+		): Promise<IResponse> => {
+			return req.url === "" || req.url === "/" || req.url.startsWith("/?")
```

Comment:
This is slightly different from the implementation of defaultRouteRequestHandler (which looks for zero path parts) - is that intentional?  I don't know offhand if it would cause any problems but was surprising to me.

Reply:
+1 to exactly copying whatever the code was before to avoid any behavior change